tensorboard/components/tensor_widget/tensor-widget-impl.ts

```diff
@@ -146,12 +178,389 @@ export class TensorWidgetImpl implements TensorWidget {
     this.infoSubsection.appendChild(shapeTagDiv);
   }
 
+  /**
+   * Fill in the content of the value divs given the current slicing spec.
+   */
+  private async renderValues() {
+    if (this.valueSection == null) {
+      this.valueSection = document.createElement('div');
+      this.valueSection.classList.add('tensor-widget-value-section')
+      this.rootElement.appendChild(this.valueSection);
+
+      this.valueSection.addEventListener('wheel', async (event) => {
```

Comment:
Can we please achieve the desired functionality without scrolljacking?
It’s extremely frustrating to interact with web sites where scrolling
diverges from the native behavior (and the native behavior differs by
platform, so this is inevitable when scrolljacking is used).

At a glance, it also looks like this is not accessible.

Reply:
I see your point, but I do think that in this UI, scroll is an easy and pleasant way to navigate the elements in the currently viewed slice of the tensor, especially when the size of the slice is large. I added a TODO item here to listen to the scroll event from the tensor widget only when one or more of the value divs are selected. The selection feature is not implemented in this PR yet.

Reply:
I'm merging the PR now. If you have any remaining thoughts or concerns, let's discuss and I can address them in PRs in the near future.